src/Falco/ConfigBuilder.fs

```diff
@@ -51,6 +54,9 @@ type ConfigBuilder (args : string[]) =
 
         bldr.AddCommandLine(args) |> ignore
 
+        if conf.AddUserSecrets then
```

Comment:
This should before env vars, I try to match the prior specified here: https://learn.microsoft.com/en-us/dotnet/core/extensions/configuration.

![image](https://github.com/pimbrouwers/Falco/assets/4595453/25083cfd-19f0-42da-aba1-62b8cc226f75)

Reply:
Goido. I'll fix that.